lua/wire/client/text_editor/texteditor.lua

```diff
@@ -2281,6 +2281,26 @@ end
 -- Runs the autocompletion
 -----------------------------------------------------------
 
+-- PickColorFind
+-- Currently works only if the main word is written completly, not if the autocompletion is used
+-- Also the tab doesn't work on the first option.
+local PickColorCV = GetConVar("wire_expression2_pickcolor_enable") or 1
```

Comment:
`or 1` here would lead to disastrous things when you try and call `PickColorCV:GetBool()`.